packages/vats/src/core/promise-space.js

```diff
@@ -50,10 +50,10 @@ export const makeStoreHooks = (store, log = noop) => {
       return;
     }
     if (store.has(name)) {
-      console.warn('cannot save duplicate:', name);
-      return;
+      store.set(name, value);
```

Comment:
how would this change get to mainnet?

"n/a" doesn't seem to suffice for upgrade considerations.

Reply:
How about adding a test in `test-promise-space.js` for this fix?

Reply:
"How do we validate this change in a testnet?" will eventually be asked. If the answer is "it's not cost-effective" then that should go in **Testing Considerations**.